apps/nextjs-website/react-components/components/Header/Header.tsx

```diff
@@ -54,15 +54,15 @@ const Header = ({
   return (
     <Box
       bgcolor={backgroundColor}
-      paddingX={{ xs: 1, sm: 3 }}
-      paddingY={3}
       component='header'
       role='banner'
+      sx={{ height: { xs: 'auto', md: 130 } }}
     >
       <Stack
         direction={{ md: 'row' }}
-        paddingY={{ xs: 2, sm: 3, md: 0 }}
-        gap={4}
+        paddingY={{ xs: 1, sm: 3, md: 0 }}
+        gap={{ xs: 2, md: 2 }}
+        style={{ height:'100%' }}
```

Comment:
```suggestion
        sx={{ height:'100%' }}
```
I think it's better to use 'sx' when working with MUI and for consistency